_data/examples.yml

```diff
@@ -158,3 +158,10 @@
   author: Martin Schönert (translated to Oscar by Thomas Breuer)
   thumbnail: oscar.png
   language: julia
+
+- title: "Generation of sporadic simple groups by $\pi$- and $\pi'$-subgroups"
```

Comment:
Actually this was not valid YAML, as it tries to interpret `\p` as a quoted character. This can be fixed by changing it to `\\p` but then the result still was not rendered as math by me. So I fixed this by using Unicode 𝜋